api/room_resource/schema.py

```diff
@@ -164,16 +164,24 @@ class Arguments:
 
     @Auth.user_roles('Admin')
     def mutate(self, info, resource_id, **kwargs):
-        query_room_resource = Resource.get_query(info)
-        active_resources = query_room_resource.filter(
+        query_resources = Resource.get_query(info)
+        active_resources = query_resources.filter(
             ResourceModel.state == "active")
-        exact_room_resource = active_resources.filter(
+        exact_resource = active_resources.filter(
             ResourceModel.id == resource_id).first()
-        if not exact_room_resource:
+        if not exact_resource:
             raise GraphQLError("Resource not found")
-        update_entity_fields(exact_room_resource, state="archived", **kwargs)
-        exact_room_resource.save()
-        return DeleteResource(resource=exact_room_resource)
+        update_entity_fields(exact_resource, state="archived", **kwargs)
```

Comment:
Good work @TeamoreA just get rid of the kwargs argument as you are not using it, also on line 166 it can be removed  from the arguments

Reply:
> Good work @TeamoreA just get rid of the kwargs argument as you are not using it, also on line 166 it can be removed from the arguments

This has been resolved @kwanj-k